include/zotonic_notifications.hrl

```diff
@@ -642,6 +642,11 @@
         state :: term()
     }).
 
+%% @doc Notify modules of a data model entry defined in manage_schema
+-record(managed_data, {
+    module :: atom(),
+    props :: list()
+}).
 
```

Comment:
Maybe call it either `datamodel_data` or `manage_data` (more active)?
I like a reference to the `#datamode{}` here, as it makes it easier to understand what this notifications does.

Also, maybe also include the whole `#datamodel{}` here (and datamodel version), as it might be useful for the manage routines to know what is happening.

/cc @arjan  @mmzeeman